packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -778,6 +778,30 @@ enum SortDirection {
   descending,
 }
 
+/// A Range-like class that works for inclusive ranges of lines in source code.
+class LineRange {
```

Comment:
can we just modify the existing Range class to support this?

e.g. add a `bool inclusive` field and create a constructor `Range.inclusive`